spec/ddtrace/benchmark/gem_loading_spec.rb

```diff
@@ -6,7 +6,7 @@
 end
 
 RSpec.describe 'Gem loading' do
-  def subject
```

Comment:
`subject` generally has semantic meaning in rspec, what's the benefit of changing the method name here?

Reply:
I think sorbet doesn't particularly like that we "override" the regular `subject` (because we're using a `def`) and gets a bit confused.

I think the change is useful anyway because `subject` and `let` in RSpec have memoization semantics -- they only get computed once per unit test, and in this spec the reason this was not a regular `subject` or `let` is because the test cases want every invocation to happen, not memoization. Thus, I think not making it seem like the regular rspec thing is actually a bonus in test readability on my book 😄